packages/addon-dev/src/rollup.ts

```diff
@@ -62,7 +90,12 @@ export class Addon {
   // their bundle names (the bundle names get generated by `publicEntrypoints`
   // above).
   output() {
```

Comment:
Why not just take options here?

Reply:
I _just_ edited the PR to ask that same question. Happy to change if that's preferred.

To answer "why not":  I didn't think of it initially :upside_down_face:

Reply:
> I just edited the PR to ask that same question

I'm really upset at github for continuing to revert my PR descriptions :cry: 
![image](https://user-images.githubusercontent.com/199018/152428764-fbbe4790-9d87-40cc-83cb-97811a98bb6d.png)

Reply:
oh it's back now. idk what's going on with github :(

Reply:
advantage of passing options to output function was too high! (changed to that approach)